lang/c/stdint_types.talon-list

```diff
@@ -0,0 +1,18 @@
+list: user.stdint_types
+code.language: c
+code.language: cpp
```

Comment:
Please double check, but I do not think that C++ actually makes use of this list.